Mathlib/Topology/Separation/Basic.lean

```diff
@@ -68,6 +69,7 @@ section Separation
 /-- A T₀ space, also known as a Kolmogorov space, is a topological space such that for every pair
 `x ≠ y`, there is an open set containing one but not the other. We formulate the definition in terms
 of the `Inseparable` relation. -/
+@[stacks 004X]
```

Comment:
```suggestion
@[stacks 004X "(2)"]
```
(And I think we refer to the numbering when there is one and fallback to "part _" only when the stacks entry is a big blob of text)